modules/fastp/main.nf

```diff
@@ -45,6 +44,7 @@ process FASTP {
             $args \\
             2> ${prefix}.fastp.log \\
         | gzip -c > ${prefix}.fastp.fastq.gz
```

Comment:
Do we still need the gzip pipe? Fastp can output gzipped files on its own

Reply:
I'm not sure.

I assumed the output piping was added because of what is said here:

https://github.com/OpenGene/fastp#output-to-stdout